.github/workflows/publish_release.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish Maven Artifacts
+
+on:
+  release:
+    types:
+      - published
+jobs:
+  bintrayUpload:
+    name: Publish Release to Maven
+    runs-on: macOS-latest
```

Comment:
Just use ubuntu-latest, I wouldn't expect macOs to be as reliable,
https://github.com/hmcts/idam-java-client/blob/master/.github/workflows/gradle.yml